ADMVEDU/ViewControllers/FilterViewController.swift

```diff
@@ -0,0 +1,242 @@
+//
+//  FilterViewController.swift
+//  ADMVEDU
+//
+//  Created by Bobr, Andrey on 6.05.21.
+//
+
+import UIKit
+
+class FilterViewController: UIViewController {
+
+    var requestOptions = RequestOptions()
+    private var stackView = UIStackView()
+    private var countryPicker = UIPickerView()
+    private var mediaPicker = UIPickerView()
+
+    private var countryTextField = UITextField()
+    private var mediaTextField = UITextField()
+
+    private var explicitSwitch = UISwitch()
+    private var limitSlider = UISlider()
+    let sliderLable = UILabel()
+    private var tableView = UITableView()
+
+    private var uniqueIdentifier = 0
+
+    weak var delegate: OptionsDelegate?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        configureStackView()
+        configureNavigationController()
+        setOptions()
+    }
+
+    func configureStackView() {
+        view.addSubview(stackView)
+        stackView.axis = .vertical
+        stackView.distribution = .fillEqually
+        stackView.spacing = 20
+        setStackViewConstraints()
+        configurePeakers()
+        configureSwitch()
+        configureSlider()
```

Comment:
Looks redundant in configureStackView method. Would be better to move it to func configureUI